penguin/tools/core/skill_tools.py

```
@@ -0,0 +1,101 @@
+        session_id: Optional[str] = None,
+    ) -> str:
+        resolved_session_id = session_id or self._current_session_id() or "default"
+        result = self.manager.deactivate(name, session_id=resolved_session_id)
+        if result.get("status") == "deactivated":
+            self._remove_skill_context(name)
+        return json.dumps(result, indent=2)
+
+    def _current_session_id(self) -> Optional[str]:
+        try:
+            conversation = self.conversation_manager.conversation
+            return conversation.session.id
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)